### ISSUE TITLE
Same "scoped" definitions in multiple scopes

### ISSUE THREAD

**Issue by shaurya947** [InsertKoinIO/koin#564]
If I have:

```
val myModule = module {
    scope(named<MyInteractor>()) {
        scoped { MyPresenter() }
    }
}
```

and

```
val myOtherModule = module {
    scope(named<MyOtherInteractor>()) {
        scoped { MyPresenter() }
    }
}
```

And I include both `myModule` and `myOtherModule` when I `startKoin()`, I get an error saying 
```
Already existing definition or try to override an existing one: [type:Scope...
```

What am I doing wrong? Intuitively, it makes sense that the same component could live in 2 unrelated scopes with different lifecycles.

**Comment by arnaudgiuliani** [comment]
Can you describe completely your case?

**Comment by arnaudgiuliani** [comment]
fixed in 2.1.0
